CONTRIBUTING.md

```diff
@@ -138,8 +138,21 @@ We generally strive to keep mdBook compatible with a relatively recent browser o
 That is, supporting Chrome, Safari, Firefox, Edge on Windows, macOS, Linux, iOS, and Android.
 If possible, do your best to avoid breaking older browser releases.
 
-Any change to the HTML or styling is encouraged to manually check on as many browsers and platforms that you can.
-Unfortunately at this time we don't have any automated UI or browser testing, so your assistance in testing is appreciated.
+GUI tests are checked with the GUI testsuite. To run it, you need to install `npm` first. Then run:
+
+```
+cargo test --test gui
+```
+
+The first time, it'll fail and ask you to install the `browser-ui-test` package. Install it then re-run the tests.
+
+If you want to disable the headless mode, use the `DISABLE_HEADLESS_TEST=1` environment variable:
+
+```
+cargo test --test gui -- --disable-headless-test
+```
+
+You can find the `browser-ui-test` package documentation on [its repository](https://github.com/GuillaumeGomez/browser-UI-test/blob/master/goml-script.md).
```

Comment:
This is missing a lot of information on how those are written. Something like this?

> The GUI tests are in the directory `tests/gui` in text files with the `.goml` extension. These tests are run using a node.js program called `browser-ui-test`. You can find documentation for this language on [its repository](https://github.com/GuillaumeGomez/browser-UI-test/blob/master/goml-script.md).

Reply:
Indeed, thanks! Hard to know what is missing when you're too involved in something. ^^'